README.md

```diff
@@ -27,31 +27,61 @@ PRs should be posted _before_ class begins on Tuesday, January 2, 2024.
 
 ### React Challenge
 
-1. React Debugging: The Card Draw React application has three (3) errors that need to be fixed. Use the console and error messages to guide you to the solution.
+1. React Debugging: The Card Draw React application has three (3) errors that need to be fixed.
+ Use the console and error messages to guide you to the solution.
+ 
+  A opening "<div>" tag was missing and needed to be added, then I removed a "return function that was not needed, then added a parenthesis for the return statment that wasnt included. Made changes but still showing faults, found the fault, was missing my export, added it back in and it runs normaly.
 
 2. React Feature: Add functionality to the `Return Cards and Shuffle Deck` button using the `shuffle` function to reset the Card Draw application to its original state.
 
-3. React Concepts: Add comments to the sever (7) notations in the `drawCard` function describing in detail the functionality of the following line of code.
+had to restore the uncommented out line the resets the hand. 
+const shuffle = () => {
+    setHand([]);
+}
+this will clear the 'hand' state when the return cards and shuffle deck is used.
 
-4. STRETCH (optional): Add stylings of your choice to the application. Ensure the styling does not interfere with the card draw functionality.
+
+3. React Concepts: Add comments to the seven (7) notations in the `drawCard` function describing in detail the functionality of the following line of code.
+
+added the answers on the app.js file
+
+
+
+4. STRETCH (optional): Add stylings of your choice to the application. Ensure the styling does not interfere with the card draw functionality. 
+
+added some border and hover stylings to the cards
 
 ### Interview Practice Questions
 
 1. What are three (3) syntax difference between HTML and JSX?
 
+ Event handlers in HTML use 'onclick','onchange' where in JSX, cammelCase naming are used in JSX while the value is produced from the function.
+
+ the class of a element is definded by the 'class' atribute in html.
+ <div class="my-contanier">Content</div>
+
+were as in JSX 'className' would be the equal attribute casue 'class; is the reserved key-word in JavaScript.
+
+< div className="my-contanier">Content</div>
+
+JSX an object accepts the 'style' object with a cammelCase property in CSS, 
+In HTML a string 'style' is typically used 
+
 2. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
 
-3. What is the difference between state values and props in React?
+In Javascript, yarn is the package manager. it was develped by facebook and is often used insted of NPM( Node Package Manager). Once Yarn is ran in the terminal is starts to look for a file called package.json. if it is in the terminal yarns start to run dependencies in the file and the creates a yarn.lock file. this will lock down version of the installed packages
```

Comment:
Nice! You are exactly right that yarn looks for a package.json and then based on what is in that file it brings in the node_modules which is the code that actually creates the functionality. Then the yarn.lock "locks" in the current versions.